### DISCUSSION TITLE
flutter example v2 - I can't handle wc: URI on mobile

### DISCUSSION THREAD

**Discussion by oleksiivinogradov** [WalletConnect/walletconnect-monorepo#2630] [Developer Support]
On desktop it works, show QR code to users. But they can't scan when working direct from mobile.
I receive:
 res: ConnectResponse(pairingTopic: d71623af45b0dc64f87f16ed5b5619ff4bf7142c3a67e05ff3deb187c16fcb1f, session: Instance of '_AsyncCompleter<SessionData>', uri: wc:d71623af45b0dc64f87f16ed5b5619ff4bf7142c3a67e05ff3deb187c16fcb1f@2?relay-protocol=irn&symKey=025519fb8ed8c426ee2977d4e131ced0f23826335896ded581fb894bd50b34d7&methods=%5Bwc_sessionPropose%2Cwc_sessionRequest%5D%2C%5Bwc_authRequest%5D)

But I have no idea what open inside mobile browser to handle uri wc:

**Comment by finessevanes** [comment]
@oleksiivinogradov can you please include the SDK version you're using, the configuration, the wallet you're using, and the repro steps please?
